docs/0.4/examples/gameroom/walkthrough/index.md

```diff
@@ -1602,103 +1606,208 @@ At the end of the operation, the `gameroom_notification` table looks like this:
+will display a notification about his invitation from Alice.
 
 #### I-3.1. Bubba Bakery UI sends authorization request to Gameroom REST API
 
 This process is the same as the Acme process in [section
 Ⅰ-1.1](#i-11-acme-ui-sends-authorization-request-to-gameroom-rest-api).
 For Bob, the general process looks like this:
 
-![](./images/auth_login_bubba1.svg "Gameroom daemon receives auth request")
+![]({% link
+docs/0.4/examples/gameroom/walkthrough/images/auth_login_bubba_1.svg %}
+"Gameroom daemon receives auth request")
 
 When Bob clicks `Log in`, the Bubba Bakery Gameroom UI hashes the password,
 then sends an authorization request to the Bubba Bakery Gameroom daemon,
-`gameroomd`. The request is handled by the Gameroom REST API, which is a part
-of `gameroomd`.
+`gameroomd`. Then, the Gameroom daemon makes some requests to the Biome REST
+API to verify the user.
 
 ```
-    POST /users/authenticate
-    {
-        email: "bob@bubbabakery.com",
-        hashedPassword: "2b944c69...c11fcf9c"
-    }
+POST /users/authenticate
+{
+  "email": "bob@bubbabakery.com",
+  "hashedPassword": "2b944c69...c11fcf9c"
+}
 ```
 
 As mentioned earlier, the UI does not reveal the user's password to the REST
 API because the password is used to encrypt signing keys.
 
-#### I-3.2. Bubba Bakery Gameroom REST API authorizes the login
+#### I-3.2. Gameroom daemon uses Biome REST API to verify password
 
-This process is the same as the Acme process in [section
-Ⅰ-1.2](#i-12-gameroom-daemon-uses-biome-rest-api-to-verify-password).
-For Bob, the general process looks like this:
+![]({% link
+docs/0.4/examples/gameroom/walkthrough/images/auth_login_bubba_2.svg %}
+"Gameroom daemon retrieves user credentials")
+
+After the Gameroom daemon has received the authentication request, the
+authentication is handled by the Biome REST API. The authentication request
+is sent from the Bubba Bakery Gameroom daemon to the Biome REST API.
 
-![](./images/auth_login_bubba2.svg "Gameroom daemon receives auth request")
+```
+POST /biome/login
+{
+  "username": "bob@bubbabakery.com",
+  "hashed_password": "2b944c69...c11fcf9c",
+}
+```
 
-When the Gameroom REST API receives the authorization request for Bob, it
-re-hashes the password sent from the browser and compares the email and hashed
-password to Bob's entry in the Bubba Bakery Gameroom database. If they match,
-authentication was successful.
+When the Biome REST API receives the authorization request for Bob, it fetches
+the entry from the Bubba Bakery Splinter daemon's local database associated with
+the username and verifies the hashed password sent in the request.
 
-The `gameroom_user` table in the Gameroom database has the following schema:
+![]({% link
+docs/0.4/examples/gameroom/walkthrough/images/auth_login_bubba_3.svg %}
+"Splinter daemon verifies Bob's credentials")
+
+The `user_credentials` table in the Splinter database has the same schema as
```

Comment:
Oops - I made a dumb mistake in my suggested rewrite.  Sorry!

It should be "Gameroom database", not "Splinter database".  (Here and in the 0.5 version.)

Reply:
The `user_credentials` table is in the Splinter database, technically, since it's a Biome related table